odoo_backup_sh/models/odoo_backup_sh.py

```diff
@@ -56,17 +59,68 @@ class BackupConfig(models.Model):
     weekly_limit = fields.Integer('Weekly limit', default=1, help='How many weekly backups to preserve.')
     monthly_limit = fields.Integer('Monthly limit', default=1, help='How many monthly backups to preserve.')
     yearly_limit = fields.Integer('Yearly limit', default=1, help='How many yearly backups to preserve.')
-    storage_service = fields.Selection(selection=[('s3', 'Amazon S3')], default='s3', required=True)
+    storage_service = fields.Selection(selection=[('odoo_backup_sh', 'Odoo Backup sh')], default='odoo_backup_sh', required=True)
+    unlimited_time_frame = fields.Char(default="hour")
+    common_rotation = fields.Selection(selection=ROTATION_OPTIONS, default='unlimited')
+    max_backups = fields.Integer(readonly=True, compute=lambda self: 0)
+    # TODO: help
+    backup_simulation = fields.Boolean(string="Demo Backup Simulation", default=False)
```

Comment:
![just-do-it](https://user-images.githubusercontent.com/186131/59765069-98657580-92b6-11e9-98ba-24efb43d12b3.jpg)